langchain4j-workers-ai/src/test/java/dev/langchain4j/model/workersai/WorkerAIChatModelIT.java

```diff
@@ -0,0 +1,74 @@
+package dev.langchain4j.model.workersai;
+
+import dev.langchain4j.agent.tool.ToolSpecification;
+import dev.langchain4j.data.message.AiMessage;
+import dev.langchain4j.data.message.ChatMessage;
+import dev.langchain4j.data.message.UserMessage;
+import dev.langchain4j.model.output.Response;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Disabled;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.condition.EnabledIfEnvironmentVariable;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import static dev.langchain4j.data.message.SystemMessage.systemMessage;
+import static dev.langchain4j.data.message.UserMessage.userMessage;
+import static dev.langchain4j.model.output.FinishReason.STOP;
+import static org.assertj.core.api.Assertions.assertThat;
+
+@Disabled("Requires a Workers ai account")
+@EnabledIfEnvironmentVariable(named = "WORKERS_AI_API_KEY", matches = ".*")
+@EnabledIfEnvironmentVariable(named = "WORKERS_AI_ACCOUNT_ID", matches = ".*")
+class WorkerAIChatModelIT {
+
+    static WorkersAiChatModel chatModel;
+
+    @BeforeAll
+    static void initializeModel() {
+        chatModel = WorkersAiChatModel.builder()
+                .modelName(WorkersAiModelName.LLAMA2_7B_FULL)
+                .accountIdentifier(System.getenv("WORKERS_AI_ACCOUNT_ID"))
+                .token(System.getenv("WORKERS_AI_API_KEY"))
+                .buildChatModel();
```

Comment:
```suggestion
                .build();
```

Reply:
The same superclass is building all models here.

Reply:
I understand the reason for this, but is just not consistent with other integrations.
You could use lombok's `@Builder` if you don't want to rewrite a lot of code